src/renderer/components/NewProjectModal/ImportMediaView.tsx

```diff
@@ -1,116 +1,136 @@
+}
 
-const ButtonContainer = styled(Box)`
-  display: flex;
-  justify-content: right;
+const CustomStack = styled(Stack)`
   width: 100%;
 `;
 
-const CustomButtonBase = styled(Button)`
-  border-radius: 0;
-  font-weight: bold;
-  color: ${colors.grey[900]};
-  text-transform: none;
-  margin: 20px 10px;
-  padding: 10px;
+const CustomColumnStack = styled(CustomStack)`
+  flex-direction: column;
 `;
 
-const CancelButton = styled(CustomButtonBase)`
-  background: ${colors.grey[400]};
-
-  &:hover {
-    background: ${colors.grey[600]};
-  }
+const CustomRowStack = styled(CustomStack)`
+  flex-direction: row;
+  align-items: center;
 `;
 
-const ActionButton = styled(CustomButtonBase)`
-  background: ${colors.lightBlue[500]};
-
-  &:hover {
-    background: ${colors.lightBlue[700]};
-  }
+const Container = styled(Box)`
+  background-color: ${colors.grey[700]};
 `;
 
-interface Props {
-  closeModal: () => void;
-  nextView: () => void;
-}
+const CustomButton = styled(Button)`
+  filter: drop-shadow(0px 8px 16px rgba(0, 0, 0, 0.8));
+`;
 
-const ImportMediaView = ({ closeModal, nextView }: Props) => {
-  const [projectName, setProjectName] = useState<string>('Example');
+const ImportMediaView = ({ prevView, closeModal, nextView }: Props) => {
+  const [isAwaitingMedia, setIsAwaitingMedia] = useState<boolean>(true);
   const [mediaFilePath, setMediaFilePath] = useState<string | null>(null);
-  const [isAwaitingProjectCreate, setAwaitingProjectCreate] =
-    useState<boolean>(false);
+  const [mediaFileName, setMediaFileName] = useState<string | null>(null);
+  const currentProject = useSelector(
+    (store: ApplicationStore) => store.currentProject
+  );
 
   const dispatch = useDispatch();
 
+  if (currentProject === null) {
+    return null;
+  }
+
+  const projectName = currentProject.name;
+
   const setCurrentProject = (project: Project) =>
     dispatch(projectCreated(project));
-
   const addToRecentProjects = async (project: Project) => {
     const projectMetadata = await retrieveProjectMetadata(project);
     dispatch(recentProjectAdded({ ...project, ...projectMetadata }));
   };
 
-  const onProjectCreate = async () => {
-    setAwaitingProjectCreate(true);
+  const handleTranscribe = async () => {
+    const project = await updateProjectWithMedia(currentProject, mediaFilePath);
 
-    const project = await makeProject(projectName, mediaFilePath);
-    if (project === null) {
-      setAwaitingProjectCreate(false);
+    if (project === null || mediaFilePath === null) {
       return;
     }
 
     setCurrentProject(project);
     await addToRecentProjects(project);
+
     nextView();
   };
 
+  const transcribeButton = (
+    <CustomButton
+      color="primary"
+      onClick={handleTranscribe}
+      disabled={isAwaitingMedia}
+      sx={{ width: '40%' }}
+    >
+      Transcribe
+    </CustomButton>
+  );
+
+  const cancelButton = (
+    <CustomButton color="secondary" onClick={prevView} sx={{ width: '40%' }}>
```

Comment:
Rather than set width 40% on both the cancel button and the transcribe button. I suggest you set the width of both to be 100% and then set the gap property (CSS) of the container element to be 32px.

This means that both buttons will try and take up as much space in their parent component as possible (so will be equal) and the gap styling will force the 32px between them.

Reply:
Done in this commit: https://github.com/patrickbrett/mlvet/pull/38/commits/db258538dce61ed27b415f8350b7af96a08cf2f0